OpenTrons/opentrons-api
# chore(monorepo): remove jest-when from repo

# Overview
vite migration branch was merged so we don't need jest-when any more. Removing jest will be done in a following pr since it will require some eslint modification.



# Test Plan
the following commands work as expect
- make check-js
- make lint-js
- make lint-css
- make test-js



# Changelog



# Review requests



# Risk assessment
low
